new durathread items

Tlaltecuhtli opened the pull request:
:cl:
add: durathread duffelbag
add: durathread toolbelt
add: durathread bandolier
fix: fixed durathread helmet to not look like a reskinned hardhat (haha)
fix: bandoliers were supposed to hold 18 but the max_combined_weightclass was set lower, this has been fixed
/:cl:

durathread stuff wasnt really worth doing and no one would ever bother asking botany to make downgraded armor or fireproof pants, this gives something that the crew might ask for as these items benefit crew more than botany alone rewarding both for interacting as they can sell these

- duffelbag : no slowdown cobby category 2 bag (by the time you get enough durathread you probably have bohs unlocked, but these are more safe to give out, less expensive and less efficient)

- toobelt : extra slots and can fit stuff like rcds, engis usually have more items available than what the can carry, this should help by creating an interaction between the 2 departments

- bandolier : cobby category 2 bandolier between trashy bag and crappy basic bandolier has 2x capacity


not sure if it was supposed to be 18 tho but i bet old coders meant it to be so else there wouldnt be a "18" written there

Tlaltecuhtli committed:
Update maps.txt

Tlaltecuhtli committed:
Update maps.txt

Tlaltecuhtli committed:
1

Tlaltecuhtli committed:
1

ATH1909 committed:
Changes the Disintegrate spell's name to "Smite" (#45786)

* Update godhand.dm

* Update touch_attacks.dm

* Update spellbook.dm

* Update spellbook.dm

* Update godhand.dm

* Update touch_attacks.dm

tgstation-server committed:
Automatic changelog generation for PR #45786 [ci skip]

Mickyan committed:
range (#45774)

tgstation-server committed:
Automatic changelog generation for PR #45774 [ci skip]

Commit:
Lowers explorer suit bullet & laser armor (#45664)

tgstation-server committed:
Automatic changelog generation for PR #45664 [ci skip]

kriskog committed:
Fixes sentience balloon icon (#45791)

Fikou committed:
bulletproof helmet add + art supply change (#45666)

Adds bulletproof helmets to cargo and completely changes artistic crates

tgstation-server committed:
Automatic changelog generation for PR #45666 [ci skip]

kriskog commented on code/game/objects/items/storage/belt.dm:
```diff
@@ -27,6 +27,47 @@
+		/obj/item/pipe_dispenser,
+		/obj/item/stack/rods,
+		/obj/item/stack/tile/plasteel,
+		/obj/item/grenade
```
I get that engineers have smart metal foam grenades, but sneaking in a general /obj/item/grenade into a 14 slot toolbelt just makes this a potential grenade belt. Narrow this down or remove it.

ATH1909 committed:
Adds robotic voiceboxes as contraband in the Robotech Deluxe (#45747)

Adds robotic voiceboxes as contraband in the Robotech Deluxe

tgstation-server committed:
Automatic changelog generation for PR #45747 [ci skip]

Tlaltecuhtli committed:
Update belt.dm

Tlaltecuhtli committed:
Update belt.dm

Akrilla committed:
Contract rerolls and general enhancements (#45703)

* refactor wip

* rerolling and qol

* space not valid

* travis be nice

* comment?

* new tablet sprites

* Start on uplink screen

* new suit sprite

* paper

* merge conflict

* Auto stash before merge of "contract-reroll" and "tgstation/master"

tgstation-server committed:
Automatic changelog generation for PR #45703 [ci skip]

Jordie0608 committed:
Deadchat ban option (#45789)

* adds deadchat ban option

* makes deadchat use new ban type

Commit:
Adds support for 3 gasses to the SM (#45676)

* Adds Tritium, Pluoxium, and BZ reactions to SM

* fixes Tritium Heat Penalty

* Makes tritium a little less deadly and more unique

* Makes the BZ Radball probability less frequent

* Makes Pluox lesser n2o

* Makes BZ less viable/op

* Makes pluoxium effective only above 15%

* Fix bad code

* Pluoxium also reduces radiation severely

* removes fuckup

* fixes gasmix ratio

* fixes some of my bad code

tgstation-server committed:
Automatic changelog generation for PR #45676 [ci skip]

MCterra10 committed:
All windows now have consistant spans (#45809)

* R-Windows now have consistant examine messages

* I forgot plasma windows too

tgstation-server committed:
Automatic changelog generation for PR #45809 [ci skip]

nemvar committed:
Gulags makes money no more. (#45817)

Bucovineanu committed:
Makes .357 speedloaders unrecyclable. (#45804)

tgstation-server committed:
Automatic changelog generation for PR #45804 [ci skip]

MMMiracles committed:
im on a boat and im going fast and (#45822)

Commit:
Wizard rounds actually check to see if the wiz died and the round is over (#45793)

tgstation-server committed:
Automatic changelog generation for PR #45793 [ci skip]

tralezab committed:
You can no longer name some things after words in the ic filter (#45696)

* quick sanity fix

* more things i found

* better list used, updated comment

carlarctg committed:
Hypospray Codersprites (#45574)

About The Pull Request

This PR adds several (terrible) unique sprites for all the combat hypos and medipens that just used the default instead, and tweaks some minor stuff like making descriptions more truthful and moving the one-use gorilla pen and combat holy water injector where they should be.

The new medipens are just recolors of the epipen, since that's the template used for the leporazine pen too (whose icon i deleted since the pen doesn't exist anymore). For example:

image
Why It's Good For The Game

These changes are intended to make each hypo stand out, so they don't all share the same sprite. It also adds a lot of inhand sprites (equally as bad in quality), so a knowledgeable player can, for example, notice they just got stabbed with a gorilla pen, or just know that they got injected with something healthy and not poison.
Changelog

cl
imageadd: added some icons and images for hyposprays and medipens so they stand out
tweak: tweaked a few descriptions on said hypos so they better describe the contents
/cl

these sprites are kind of terrible, but i think they're better than every single pen and combat hypo using the same icon. any new sprites would be highly appreciated, especially for these two

tgstation-server committed:
Automatic changelog generation for PR #45574 [ci skip]

ExcessiveUseOfCobblestone committed:
Dissection Examine + Logic Fix + Ayy (#45807)

clDissectby
add: Examining a dissected body will provide insight on the highest tier performed.
fix: You can do ayy dissection now if you get the tech
fix: FINALLY fixed the surgery logic correctly
/cl

tgstation-server committed:
Automatic changelog generation for PR #45807 [ci skip]

nemvar committed:
Fixes bulldog inhands (#45833)

* The bulldog has inhands once more.

* forgot these are a thing

tgstation-server committed:
Automatic changelog generation for PR #45833 [ci skip]

tgstation-server committed:
Automatic changelog compile, [ci skip]

silicons committed:
Update view_variables.dm (#45856)

tgstation-server committed:
Automatic changelog generation for PR #45856 [ci skip]

jdawg1290 committed:
Close quarantine module loophole (#45766)

* Quarantine module now has teeth

* Quarantine applies to everyone

* typofix and more genericness

* Legal clarification

* Rules lawyering

* Rule 1 friendlier version

tgstation-server committed:
Automatic changelog generation for PR #45766 [ci skip]

kriskog committed:
Selected Access in HoP Console is Now Green (From Red) For Viewability (#45848)

* Removes red/blue ID console button combo.

* Different color.

tgstation-server committed:
Automatic changelog generation for PR #45848 [ci skip]

penterwast committed:
logging for binary chat (#45841)

About The Pull Request

It is currently impossible to tell from logs whether a silicon said a message over binary chat or said it out loud. This PR marks binary chat messages with (binary) in say logs.
Why It's Good For The Game

Removes ambiguity from log files, makes investigating easier.
Changelog

cl Penterwast
admin: Binary chat messages are now marked as such in logs.
/cl

tgstation-server committed:
Automatic changelog generation for PR #45841 [ci skip]

nemvar committed:
Doors no longer update_icon in Destroy (#45830)

nemvar committed:
Inherent_factions for species. (#45828)

* Inherent_factions for species.

* forgot to actually remove this.

* oh it actually got used somewhere but was redundant

tgstation-server committed:
Automatic changelog generation for PR #45828 [ci skip]

actioninja commented:
do we really need even more storage powercreep?

collinlunn committed:
Improvised shotgun shell recipe simplification (#45792)

* improv shotgun shell recipe change

* improv shell craft time

* Update recipes.dm

* Update recipes.dm

* Update recipes.dm

AcapitalA committed:
Removes bolt of death from xenobiology magicarps. (#45800)

* Removes bolts of death from all magicarps.

The bolt of death has been removed from the Magicarp and Chaos Magicarp.

* Only xenobiology carps are affected.

Wizard event carps still got their bolts of death.

* Gives xenobiology magicarps the correct colors.

Fixes a would-be bug where xenobiology magicarps would have normal carp colors. - I really shouldn't have made a commit at 2AM while being tired as hell...

* Removes call with no recieving end. (probably)

Removed a ..() which led to nowhere.

* Adds empty space, removes a tab.

Removes a tab at 82. - Oops.

* Removes another call going nowhere.

Removed another ..(), but for the chaos variant this time. - Travis please I'm getting worried.

* Cleans up code a little.

Compressed code a bit more. - God I love redefines.

* Compresses the code even more.

Since procs inherit I managed to save some more space. - Thanks cobb.

* Stops tracker from tracking nonexistent changes.

It's the same list as before this PR now.

tgstation-server committed:
Automatic changelog generation for PR #45800 [ci skip]

81Denton committed:
Laserpointer scaling fix (#45860)

h

tgstation-server committed:
Automatic changelog generation for PR #45860 [ci skip]

Akrilla committed:
my fuckup (#45827)

actioninja committed:
Reinforced window construction fixes (#45890)

About The Pull Request

Fixes a couple of issues related to window construction introduced by the tougher windows PR.
Also gives plastitanium and shuttle windows matching values to reinforced and plasmawindows like I intended to on the last PR.
Why It's Good For The Game

fix
Changelog

cl
fix: newly constructed reinforced windows start in the correct state, meaning they can be disassembled
fix: crowbar can no longer be used out of order on construction
fix: screwed down but not levered in windows can be unscrewed again
tweak: shuttle and plastitanium windows now have the proper values to match reinforced windows.
/cl

tgstation-server committed:
Automatic changelog generation for PR #45890 [ci skip]

Tlaltecuhtli commented:
there are no intermediates between roundstart and lolBoH for storage,
having as engi not pick immersive tools like the holo projector because its a waste of space is bad
and having botany actually help crew is always good

Tlaltecuhtli committed:
e

Tlaltecuhtli committed:
Merge branch 'durathreadss' of https://github.com/Tlaltecuhtli/tgstation into durathreadss

Tlaltecuhtli commented:
ah fuck

ExcessiveUseOfCobblestone commented:
> cobby category 2 bandolier 

nani

kriskog commented:
> 
> 
> > cobby category 2 bandolier
> 
> nani

Yeah this is clearly a cat 3

ExcessiveUseOfCobblestone commented:
gonna have to nerf it then, too many cat 3 bandoliers.
